demo/demo_image_repo.py

```diff
@@ -26,7 +26,6 @@
 """
 from __future__ import print_function
 from __future__ import unicode_literals
-from io import open
```

Comment:
I'll try removing them locally and run tests and the demo and if it works, they'll come in a future commit.